pkg/bundle/resources.go

```diff
@@ -332,6 +332,14 @@ func readContent(ctx context.Context, progress *progress.Progress, base, name st
 		}
 		httpGetter.Client.Transport = transport
 	}
+	if auth.SshPrivateKey != nil {
+		if strings.IndexAny(c.Src, "&;") == -1 {
+			c.Src += "?"
+		} else {
+			c.Src += "&"
+		}
```

Comment:
I understand how this code works, but why do we need this check? Checked the SSH (rsa256) spec and am curious.

Reply:
This is just trying to `sshkey` query parameters to url without parsing it. It is how https://github.com/hashicorp/go-getter#git-git reads it. Parsing the url is tricky as it is git protocol so it requires special. So instead I just detect the query parameters and add it at the end of string.

Reply:
Good idea. Parsing would have more edge cases, so I agree with the greedy detection and addition.